$NetBSD$

CSS grid, box alignment keywords, gradients, border radii and shadows: parse, cascade and compute them.

--- src/parse/properties/Makefile.orig
+++ src/parse/properties/Makefile
@@ -51,6 +51,8 @@ DIR_SOURCES :=				\
 	font.c				\
 	font_family.c			\
 	font_weight.c			\
+	grid.c				\
+	background_image.c		\
 	list_style.c			\
 	list_style_type.c		\
 	margin.c			\
